build/Jenkinsfile.pr

```diff
@@ -236,12 +236,38 @@ pipeline {
             }
         }
 
+        stage('Clean checkout') {
+            // We need to perform a clean checkout here to ge the original git commit hash from the PR
```

Comment:
Just to be clear, is this a "clean" checkout, removing generated files like `report*.xml`, or just intending to checkout a particular commit?

Reply:
just to checkout to get the commit hash. will not touch in any other file, also this checkout is being made in another folder